src/rp2_common/hardware_clocks/scripts/vcocalc.py

```diff
@@ -5,11 +5,13 @@
 parser = argparse.ArgumentParser(description="PLL parameter calculator")
 parser.add_argument("--input", "-i", default=12, help="Input (reference) frequency. Default 12 MHz", type=float)
 parser.add_argument("--vco-max", default=1600, help="Override maximum VCO frequency. Default 1600 MHz", type=float)
-parser.add_argument("--vco-min", default=400, help="Override minimum VCO frequency. Default 400 MHz", type=float)
+parser.add_argument("--vco-min", default=750, help="Override minimum VCO frequency. Default 750 MHz", type=float)
 parser.add_argument("--low-vco", "-l", action="store_true", help="Use a lower VCO frequency when possible. This reduces power consumption, at the cost of increased jitter")
 parser.add_argument("output", help="Output frequency in MHz.", type=float)
 args = parser.parse_args()
 
+assert((args.vco_min >= 750) and (args.vco_max <= 1600))
```

Comment:
Can we remove this assert please. The limits are set and checked by default, but I don't see any reason to stop people from raising limits they want to. We're not cops.

Reply:
Yes, which checks against the limits set by PICO_CONFIG_ defines. This assert was checking the limits themselves, which is different.

Reply:
> Hmmm, so should the `PICO_PLL_VCO_` defines get turned into "full" PICO_CONFIG defines by adding the appropriate comment-lines like e.g. https://github.com/raspberrypi/pico-sdk/blob/master/src/rp2_common/pico_platform/include/pico/platform.h#L24 ?

the reason i didn't is that i didn't expect many people would need to (i was going to include it in the release notes)